furiosa/models/_utils.py

```diff
@@ -27,11 +29,21 @@
 module_logger = logging.getLogger(__name__)
 
 
-def get_version_info() -> Optional[str]:
-    from furiosa.tools.compiler.api import VersionInfo
-
-    version_info = VersionInfo()
-    return f"{version_info.version}_{version_info.git_hash}"
+def get_version_info() -> str:
+    try:
+        compiler_info = subprocess.run(
+            ['furiosa-compiler', '--version'], capture_output=True, text=True, check=True
+        ).stdout
+    except FileNotFoundError:
+        raise RuntimeError("furiosa-compiler is not installed")
+    pattern = r"frontend:\n- version: (.+?)\n- revision: (.+?)\b"
```

Comment:
backend instead of frontend?

Reply:
Thanks This part was written before our meeting, so I made a mistake

Reply:
Updated in https://github.com/furiosa-ai/furiosa-models/compare/d16a8d7e7e1ecb4722b238324194ec13ba1279aa..c6eaaccc98da57378ec10c6e4902d5fc6b2b99b3#diff-8791739779fb36bc016fe872dc823075b870b06fc43a2d232928df4c9fa1aacfL39